File: base64_and_file/src/main/java/com/phone/base64_and_file/IBase64AndFileView.java
package com.phone.base64_and_file;

import com.phone.base64_and_file.bean.Base64AndFileBean;
import com.phone.common_library.base.IBaseView;

public interface IBase64AndFileView extends IBaseView {

    void showCompressedPictureSuccess(Base64AndFileBean success);

    void showCompressedPictureError(String error);

    void showPictureToBase64Success(Base64AndFileBean success);

    void showPictureToBase64Error(String error);

    void showBase64ToPictureSuccess(Base64AndFileBean success);

    void showBase64ToPictureError(String error);

}

File: library_base64_and_file/src/main/java/com/phone/library_base64_and_file/view/IBase64AndFileView.kt
package com.phone.library_base64_and_file.view

import com.phone.library_base64_and_file.bean.Base64AndFileBean
import com.phone.library_base.base.IBaseView

interface IBase64AndFileView : IBaseView {

    fun showCompressedPictureSuccess(success: Base64AndFileBean)

    fun showCompressedPictureError(error: String)

    fun showPictureToBase64Success(success: Base64AndFileBean)

    fun showPictureToBase64Error(error: String)

    fun showBase64ToPictureSuccess(success: Base64AndFileBean)

    fun showBase64ToPictureError(error: String)

}